src/components/opt-in-flyout/opt-out-reason-selector.js

```diff
@@ -68,6 +73,7 @@ class OptOutReasonSelector extends composeMixins(
 
 	render() {
 		return html`
+			<label for="selector">${this.localize('components:optInFlyout:feedbackReasonLabel')}</label>
```

Comment:
Should this be getting the `d2l-input-label` class like the core input labels do?

Reply:
So you can see what this actually looks like:

<img width="686" alt="Screenshot 2023-08-21 at 2 50 45 PM" src="https://github.com/BrightspaceUI/labs/assets/5491151/c9b3d973-7d75-4a95-813d-f09ca6df927d">

The answer is "probably"? I started going down that path, and then converting the `<d2l-input-textarea>` to also use our built-in labels, but they end up being `0.8rem` instead of `1rem` so visually things look quite different. As I mentioned in the PR description, I didn't really feel comfortable changing the visual style as part of what should just be a migration to the monorepo.

Reply:
Yep, fair enough.